src/software/sensor_fusion/filter/ball_filter.h

```diff
@@ -43,6 +43,8 @@ class BallFilter
     static constexpr double MAX_BUFFER_SIZE_VELOCITY_MAGNITUDE = 4.0;
     // The extra amount beyond the ball's max speed that we treat ball detections as valid
     static constexpr double MAX_ACCEPTABLE_BALL_SPEED_BUFFER = 2.0;
+    // The maximum error threshold to considering using the generated linear regression
+    static constexpr double LINEAR_REGRESSION_ERROR_THRESHOLD = 1000.0;
```

Comment:
Do you know how we came up with this value?

Reply:
This is just a value that jon used during robocup, do you think its worth it to look into adjusting this value

Reply:
I'm not sure, we could probably keep it and tune it if we end up finding that our ball filter needs improvements. @itsarune what do you think?

Reply:
Actually just noticed that you tagged #2752 to it which is probably the right thing to do

Reply:
Hmm not sure what this constant is referring to me, but we can tune and adjust later